cmd/mt-index-cat/main.go

```diff
@@ -74,13 +78,22 @@ func main() {
 		fmt.Printf("output: or custom templates like '{{.Id}} {{.OrgId}} {{.Name}} {{.Metric}} {{.Interval}} {{.Unit}} {{.Mtype}} {{.Tags}} {{.LastUpdate}} {{.Partition}}'\n\n\n")
 		fmt.Println("You may also use processing functions in templates:")
 		fmt.Println("pattern: transforms a graphite.style.metric.name into a pattern with wildcards inserted")
+		fmt.Println("         an operation is randomly selected between: replacing a node with a wildcard, replacing a character with a wildcard, and passthrough")
+		fmt.Println("patternCustom: transforms a graphite.style.metric.name into a pattern with wildcards inserted according to rules provided:")
+		fmt.Println("         patternCustom <chance> <operation> [,<chance> <operation>, ...]")
+		fmt.Println("         the chances need to add up to 100")
+		fmt.Println("         operation is one of:")
+		fmt.Println("         pass         (passthrough)")
+		fmt.Println("         <number>rcnw (replace number random consecutive nodes with wildcards")
```

Comment:
that's a confusing sentence `replace number random consecutive nodes with wildcards`. I think i understand how it's meant, but hard to read it a few times.

Reply:
how about `replace <number> random consecutive nodes with wildcards` ?

Reply:
yeah, that would be much clearer because the `number` doesn't look like a word in the sentence